index.js

```diff
@@ -5,11 +5,12 @@ const fs = require('fs');
 
 function run() {
   const lcovPath = core.getInput('path');
-  const minCoverage = core.getInput('min_coverage');
+  const minCoverageInput = core.getInput('min_coverage');
   const excluded = core.getInput('exclude');
   const excludedFiles = excluded.split(' ');
+  const minCoverage = parseMinCoverage(minCoverageInput);
 
-  if (!canParse(lcovPath)) {
+  if (minCoverage === null || !canParse(lcovPath)) {
     return;
   }
```

Comment:
So, should `'10%'` be parsed as `10`? I thought it should not, based on [this comment](https://github.com/VeryGoodOpenSource/very_good_coverage/issues/255#issuecomment-1607562649).

Reply:
True. I was personally not so concerned about allowing `10%`. However, now that you bring attention to it, it might give the false illusion that if that works then `25 / 200` might also work for example. I guess we're safer keeping as Number only.  

Feel free to amend the above to:
```js
const minCoverage =
    minCoverageInput.trim().length === 0 ? 100 : Number(minCoverageInput);
```